legacy/build.gradle

```diff
@@ -15,15 +15,19 @@ dependencies {
     compile project(":cdm")
     compile project(":grib")
     compile project(":opendap")
-    
+
     compile libraries["guava"]
     compile libraries["jdom2"]
     compile libraries["jsr305"]
-    compile libraries["aws-java-sdk-s3"]  // For CrawlableDatasetAmazonS3.
+    compile libraries["awssdk-s3"]  // For CrawlableDatasetAmazonS3.
     compile libraries["jackson-core"]  // Replace what was in aws-java-sdk-s3
     compile libraries["jackson-annotations"]  // Replace what was in aws-java-sdk-s3
     compile libraries["jackson-databind"]  // Replace what was in aws-java-sdk-s3
-    
+
+    compile libraries["awssdk-sdk-core"]
+    compile libraries["aws-sdk-core"]
+    compile libraries["aws-sdk-s3"]
+
```

Comment:
Are these all libs that Brian used and you figured it out?

Reply:
Those are developer-chosen names that ref a library definition in another file. Lemme see which one...